Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.8 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.8 AS builder
```

Comment:
TIL we still have this Dockerfile and it was missed when we were separating out the downstream builds. Maybe this is something we can rework in the future to avoid using non-external images? cc @joelanford

Reply:
Yeah, in general I think there's a bunch of cleanup of Dockerfiles and quay build triggers we can do in this repo now.

Do we need anything other than the `quay.io/operator-framework/opm` image at this point?